code/modules/racial_roundstartperks/racial_perk.dm

```diff
@@ -0,0 +1,253 @@
+			continue
+
+		H.reagents?.add_reagent("amatoxin", 5)
+		H.emote("gasp")
+		to_chat(H, SPAN_WARNING("[holder] shudders"))
+
+/datum/perk/overworkedspeed/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your legs ache, you'll need more time to recover before using this again."))
+		return FALSE
+	cooldown_time = world.time + 15 MINUTES
+	user.visible_message("[user] begins breathing much quicker as they let out a gasp!", "You feel your heart rate increasing rapidly as everything seems to speed up and you let out an excited gasp!", "You hear a loud gasp...")
+	playsound(usr.loc, 'sound/effects/mob_effects/gasp_f1.ogg', 50, 1)
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("satakol", 5)
+	return ..()
+
+
+/datum/reagent/medicine/satakol
+	name = "Satakol"
+	id = "satakol"
+	description = "satakol is a highly effective, short term, muscle stimulant naturally produced by several species when under stress."
+	taste_description = "acid"
+	reagent_state = LIQUID
+	color = "#FF3300"
+	nerve_system_accumulations = 45
+	appear_in_default_catalog = FALSE
+	constant_metabolism = TRUE
+	scannable = TRUE
+
+/datum/reagent/medicine/satakol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.stats.addTempStat(STAT_TGH, 5, STIM_TIME, "satakol")
+	M.stats.addTempStat(STAT_ROB, 5, STIM_TIME, "satakol")
+	M.add_chemical_effect(CE_SPEEDBOOST, 0.2)
+	M.add_chemical_effect(CE_PULSE, 2)
+
+/////////////////////////////////////////Cindarite based perks
+/datum/perk/purgemaluses
+	name = "Purge Maluses"
+	desc = "You force your body to begin the process of removing toxins from your blood. All toxins, addictions, and stimulants are slowly purged while any toxin damage to your liver or body is healed but the effect leaves you exhausted.... and aswell making your body way more vunerable to toxins."
+	active = FALSE
+	passivePerk = FALSE
+
+/datum/perk/purgemaluses/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your body aches with the pain of its recent purge, you'll need more rest before using this effect again."))
+		return FALSE
+	cooldown_time = world.time + 15 MINUTES
+	user.visible_message("[user] shivers slightly as they begin to slow down.", "You start to feel quite chilly and tired as your body begins purging toxins within your blood.")
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("sataricillin", 5)
+	return ..()
+
+/datum/perk/purgemaluses/assign(mob/living/carbon/human/H)
+	..()
+	holder.toxin_mod_perk += 0.1 //Might be to high...
+
+/datum/perk/purgemaluses/remove()
+	holder.toxin_mod_perk -= 0.1
+	..()
+
+
+/datum/reagent/medicine/spaceacillin/sataricillin
```

Comment:
This whole medicine is acting 1:1 with spaceacillin and does not purge any toxins, nor stimulants, nor reduces your toxloss in any way, this is just a 5u shot of Spaceacillin. I suggest adding its own affect_blood() proc with the combined effects of:
**Purger**
 `M.add_chemical_effect(CE_PURGER, 1)`
**Haloperidol since it removes Toxins and Stims:**
```	if(M.bloodstr)
		for(var/current in M.bloodstr.reagent_list)
			var/datum/reagent/toxin/R = current
			if(!istype(R, src))
				R.remove_self(effect_multiplier * pick(list(2,4,6)))
			var/datum/reagent/stim/S = current
			if(!istype(S, src))
				S.remove_self(effect_multiplier * pick(list(2,4,6)))
```
**And some minor Dylovene effect:**
`M.adjustToxLoss(-((0.1 + (M.getToxLoss() * 0.05)) * effect_multiplier))`

As well as upping the cooldown to 30 minutes for the perk as suggested earlier since that'd make it have the desired effects and would be too good to spam every fifteen.